client/layout/index.jsx

```diff
@@ -33,6 +33,7 @@ import PulsingDot from 'components/pulsing-dot';
 import SitesListNotices from 'lib/sites-list/notices';
 import OfflineStatus from 'layout/offline-status';
 import QueryPreferences from 'components/data/query-preferences';
+import TracksOptOut from 'layout/tracks-opt-out';
```

Comment:
Is this component needed in layout? Could it be moved to the user settings page?

Reply:
The idea is to handle the case when someone is logged in browser A and updates the setting in browser B or in a mobile app. At the next reload of Calypso in browser A, the user settings will be fetched and the cookie toggled accordingly. If we put this in the user settings page or in the login page, the opt-out wouldn't be effective for this user on browser A until he logs back again ( or until he visits the user settings again )